rpadmin/adminv2.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2025 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package rpadmin
+
+import (
+	"context"
+
+	"buf.build/gen/go/redpandadata/core/connectrpc/go/redpanda/core/admin/v2/adminv2connect"
+	"connectrpc.com/connect"
+)
+
+// BrokersService returns a client for the BrokerService of the Admin API V2.
+func (a *AdminAPI) BrokersService(opts ...connect.ClientOption) adminv2connect.BrokerServiceClient {
+	opts = append([]connect.ClientOption{
+		connect.WithInterceptors(newContentTypeInterceptor()),
```

Comment:
Don't love adding this to every service, but let me know if you have a better way. thanks!